clang/tools/clang-sycl-link-wrapper/SYCLLinkOpts.td

```diff
@@ -0,0 +1,50 @@
+// We try to create options similar to lld's.  That way, options passed to clang
+// -Xoffload-linker can be the same whether offloading to nvptx or amdgpu.
```

Comment:
Copy-paste from another source? I expect this comment reference spir-v instead of nvptx and amdgpu.

Reply:
Yes. Sorry. I thought I had cleaned it up.